src/PollinationSDK/Model/S3Source.cs

```diff
@@ -38,25 +38,25 @@ public partial class S3Source :  IEquatable<S3Source>, IValidatableObject
         /// <summary>
         /// Initializes a new instance of the <see cref="S3Source" /> class.
         /// </summary>
-        /// <param name="type">type (default to &quot;s3&quot;).</param>
-        /// <param name="key">The path inside the bucket to source artifacts from. (required).</param>
-        /// <param name="endpoint">The HTTP endpoint to reach the S3 bucket. (required).</param>
         /// <param name="bucket">The name of the S3 bucket on the host server. (required).</param>
         /// <param name="credentialsPath">Path to the file holding the AccessKey and SecretAccessKey to authenticate to the bucket. Assumes public bucket access if none are specified..</param>
+        /// <param name="endpoint">The HTTP endpoint to reach the S3 bucket. (required).</param>
+        /// <param name="key">The path inside the bucket to source artifacts from. (required).</param>
+        /// <param name="type">type (default to &quot;s3&quot;).</param>
         public S3Source
         (
-           string key, string endpoint, string bucket, // Required parameters
-           string type = "s3", string credentialsPath= default// Optional parameters
+           string bucket, string endpoint, string key, // Required parameters
+           string credentialsPath= default, string type = "s3"// Optional parameters
```

Comment:
Type "s3" doesn't match class name "S3Source". Is this from Queenbee? 
cc @mostaphaRoudsari